RELEASE.md

```diff
@@ -3,7 +3,7 @@
 ## Overview 
 Release are created from the main repository branch using the version
 numbers, including an intermediate version suffix, 
-defined in `.make.versions`.
+defined in `.make.versions`
 The following points are important:
 
 1. In general, common a version number is used for all published pypi wheels and docker images.
```

Comment:
do you mean: a common (it is written common a)

Reply:
Done. Thanks.